src/Tools/MicroBuild/publish-assets.ps1

```diff
@@ -32,6 +32,25 @@ try
 {
     Write-Host "Starting PostTest script..."
 
+    # We need to remove 'refs/heads/' from the beginning of the string
+    $branchName = $branchName -Replace "^refs/heads/"
+    
+    # We also need to replace all instances of '/' with '_'
+    $branchName = $branchName.Replace("/", "_")
+
+    switch ($branchName)
+    {
+        "dev15-rc" { } 
+        default
+        {
+            if (-not $test)
+            {
+                Write-Host "Branch $branchName is not supported for publishing"
```

Comment:
@jasonmalinowski 

> it should still build fine. It just shouldn't publish. 

In VSTS build == publish.  There is no distinction between them.  I'm unsure what point you are trying to make here.